vector/src/main/java/im/vector/activity/VectorRoomActivity.java

```diff
@@ -1452,6 +1452,13 @@ public void onSendAnyway() {
         });
     }
 
+    @Override
+    public void onConsentNotGiven(Event event, MatrixError matrixError) {
```

Comment:
I don't find any usage of this method. Is it normal?

Reply:
Yes, it's called from the SDK